grpchealth/grpchealth.go

```diff
@@ -20,7 +20,7 @@ type HealthCheckingService struct {
 	grpc_health_v1.UnimplementedHealthServer
 	sync.RWMutex
 	healthStatus map[string]grpc_health_v1.HealthCheckResponse_ServingStatus
-	log Logger
+	log          Logger
```

Comment:
is that indentation correct?

Reply:
well yes it is - 

I have also added a gofmt cmd to the task qa-basic